presto-docs/src/main/sphinx/develop/client-protocol.rst

```diff
@@ -0,0 +1,168 @@
+===========================
+Presto Client HTTP Protocol
```

Comment:
I would rename "HTTP Protocol" to "REST API".

Reply:
Renamed.